### ISSUE TITLE
how to deal the id = -1 in training data

### ISSUE THREAD

**Issue by tangsipeng** [ifzhang/FairMOT#159]
i don't quite understand how to deal the id = -1 in training data ？

**Comment by ifzhang** [comment]
These data do not have id label, so the id is set -1. When computing the id loss, we use cross entropy loss, which will ignore the -1 label. This means we do not compute id loss for those datasets without id label.

**Comment by tangsipeng** [comment]
> 
> 
> These data do not have id label, so the id is set -1. When computing the id loss, we use cross entropy loss, which will ignore the -1 label. This means we do not compute id loss for those datasets without id label.

i got it, thanks your reply
